OWL2/OWL22CommonLogic.hs

```diff
@@ -257,7 +258,7 @@ mapMap :: Map.Map Entity IRI -> Result (Map.Map Id Id)
 mapMap m = return $ Map.map uriToId $ Map.mapKeys entityToId m
 
 mapSymbol :: Entity -> Set.Set Symbol
-mapSymbol (Entity _ _ iri) = Set.singleton $ idToRaw $ uriToId iri
+mapSymbol (Entity _ _ iRi) = Set.singleton $ idToRaw $ uriToId iRi
```

Comment:
Why the unconventional `iRi` instead of `iri` (throughout the file)?

Reply:
Because it conflicted with the method iri from Common.IRI. I have renamed it to iriParser and now we can use iri. Have replaced it.